pom.xml

```diff
@@ -361,6 +361,7 @@
           <filterProperties>
             <repository.proxy.url>${repository.proxy.url}</repository.proxy.url>
           </filterProperties>
+          <invokerTest>it-abstract-versions-report-001</invokerTest>
```

Comment:
😄 
you can use property  on cmd `mvn ... -Dinvoker.test=xxx`

Reply:
Done.

Reply:
Done.